docs/changelog.md

```diff
@@ -27,6 +27,9 @@ Added:
   next release, rather than only Python 3.10 (!377).
 - [Scan.group_data] method to make an xarray or pandas GroupBy object based on
   scan steps (!379).
+- [fit_gaussian()][extra.utils.fit_gaussian] has a new `nans_on_failure`
+  argument to allow returning NaN arrays when fitting fails, and now also
+  supports [DataArray][xarray.DataArray] inputs (!385).
```

Comment:
I'd maybe put these as separate bullet points, since they seem to be orthogonal changes that you just happened to make together.